tests/performance_tests/test_dark_storage_performance.py

```diff
@@ -171,9 +171,9 @@ def test_direct_dark_performance_with_storage(
 ):
     key = {
         "summary": "PSUM1",
-        "gen_data": "POLY_RES_1@0",
+        "gen_data": "POLY_RES_1",
```

Comment:
What happens to the report step in plotting now? Not that I like the old syntax, but is it gone in this PR?

Reply:
Seems that putting the `@0` back still works, so putting it back

(in https://github.com/equinor/ert/pull/7563)